src/lib/zf/log.c

```diff
@@ -135,14 +136,25 @@ void zf_log(struct zf_stack* st, const char* fmt, ...)
   va_end(va);
 }
 
+void zf_log_set_file_name(const char *file)
```

Comment:
looks like only used in this file so should be static

Reply:
yes, thanks.